internal/pkg/aws/ec2/ec2.go

```diff
@@ -135,7 +135,6 @@ func extractResource(label string) (*Resource, error) {
 		return nil, fmt.Errorf("extract resource ID from string: %s", label)
 	}
 	splitResource := strings.SplitN(label, " ", 2)
-	// TODO: switch to regex to make more robust
```

Comment:
"[regex is overrated](https://github.com/aws/copilot-cli/pull/3823/commits/12b55d58e58d1c6d1e348e68eef9e04c8ec3084d)" 🤣 (not saying I disagree)